src/chapters/angular-lsn3/attribute-directives.rst

```diff
@@ -0,0 +1,224 @@
+
+      h3 {
+         text-align: center;
+         text-decoration: underline;
+      }
+
+      .skills {
+         color: green;
+      }
+
+   ``skill-set.component.html``
+
+   .. sourcecode:: html+ng2
+      :linenos:
+
+      <div class="skills">
+         <h3>{{listHeading}}</h3>
+         <ol>
+            <li *ngFor="let skill of skills">{{skill}}</li>
+         </ol>
+         <hr>
+         <h3>Copy of Skill List</h3>
+         <ol>
+            <li *ngFor="let skill of skills">{{skill}}</li>
+         </ol>
+         <hr>
+         <p>Here is some practice text...</p>
+      </div>
+
+   ``skill-set.component.ts`` SkillSetComponent class:
+
+   .. sourcecode:: TypeScript
+      :linenos:
+
+      export class SkillSetComponent implements OnInit {
+         listHeading: string = 'Some Coding Skills I Know';
+         skills: string[] = ['Loops', 'Conditionals', 'Functions', 'Classes', 'Modules', 'Git', 'HTML/CSS'];
+         alternateColor: string = 'black';
+         bulletType: string = 'A';
+         changeColor: boolean = true;
+
+         constructor() { }
+
+         ngOnInit() { }
+
+      }
+
+Right now, there is an awful lot of green on the page, which is set by the
+``skills`` class in the CSS file. Let's fix this with some attribute
+directives.
+
+Inline Styling
+^^^^^^^^^^^^^^^
+
+To change the color and bullet type of the first list element, we could
+override the CSS instructions with some inline code:
+
+.. sourcecode:: html
+
+   <ol style="color: black" type="A">
+
+However, we can use what we learned about data-binding to replace these
+hard-coded styles with variables:
+
+.. sourcecode:: html+ng2
+
+   <ol [style.color]="alternateColor" [type]="bulletType">
+
+Ideas to note:
+
+#. Unlike the structural directives ``*ngFor`` and ``*ngIf``, we can add more
+   than one attribute directive to an HTML tag.
+#. The ``style`` attribute has different properties that can be assigned using
+   dot notation. Examples include ``style.color`` and ``style.background``.
+#. The variables ``alternateColor`` and ``bulletType`` are assigned in
+   ``skill-set.component.ts`` file.
+#. NEAT! Reassigning the ``alternateColor`` variable in the ``.ts`` file
+   causes EVERY tag with ``[style.color]="alternateColor"`` to change color.
+
+.. admonition:: Try It
+
+   Change the values for the ``alternateColor`` and ``bulletType`` variables.
+   Save your work and refresh the webpage to see the results.
+
+   Note that ``bulletType`` takes options of numbers (``1``), upper and lower
+   case letters (``A``, ``a``), or upper and lower case Roman numerals (``I``,
+   ``i``).
+
+Changing Styles with Booleans
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can accomplish the same results by applying a class to the second ``ol``
+tag:
+
+#. Add the following code to ``skill-set.component.css``:
+
+   .. sourcecode:: html
+      :linenos:
+
+      .ol-style {
+         color: black;
+         text-align: center;
+         list-style-type: upper-roman;
```

Comment:
I worried about that as well.  We should probably just rename the class.  Suggestions?

Reply:
the 'ol-style' class name wasn't confusing to me. it was the fact that the css property 'list-style-type' is controlling the same thing as the html attribute 'type'. we can't change either of those names but it could be worth clarifying that this is a different way to go about changing the style of the list? ie, 
"Note, the css property 'list-style-type' defines the look of the list item markers, similar to the html element ol's 'type' attribute. The values available to the css property are different, however. You can find a full list here: https://www.w3schools.com/cssref/pr_list-style-type.asp

Reply:
Nice approach.  I used your phrasing in a Note admonition after this step.